tests/test_integration.py

```diff
@@ -165,3 +166,12 @@ def test_downgrade_to_old_version(db):
     ids = db.tests.insert([newer_document])
     obj = db.tests.find_one(ids[0])
     assert obj.foo == 'downgraded foo'
+
+
+@pytest.mark.skipif(not PY3, reason='Int64 requires pymongo3')
```

Comment:
Pymongo 3.x is also for python 2.x. I'd rather you've changed the skipping condition. For example:

```
def test_insert_int64(db):
     try:
         from bson.int64 import Int64
      except ImportError:
           pytest.skip('old pymongo version doesn\'t have Int64 class')
```